src/core/inlines.js

```diff
@@ -115,6 +115,15 @@ function inlineVariableMatches(matched) {
   return hyperHTML`<var data-type="${type}">${varName}</var>`;
 }
 
+function inlineLinkMatches(matched) {
+  const parts = matched
+    .slice(2, -2) // Chop [= =]
+    .split(/\/(?:\s?)/) // Only at first "/"
+    .map(s => s.trim());
```

Comment:
If we just want to split at only the first `/`, we can do`.split("/", 2)`. Are there any pitfalls?

Reply:
no pitfalls, I'd just forgotten about that second argument